tests/unit/metadata/bucketclient/backend.js

```diff
@@ -0,0 +1,89 @@
+const assert = require('assert');
+const http = require('http');
+const BucketClientInterface =
+    require('../../../../lib/metadata/bucketclient/backend').default;
```

Comment:
default?

Reply:
Nice catch. Didn't see that when trying to forward 6.4 to 7.0